netutils/data_files/oui_mappings.py

```diff
@@ -6523,103 +6522,104 @@
     "a4c7f6": "Extreme Networks, Inc.",
     "cc00f1": "Sagemcom Broadband SAS",
     "d46352": "Vutility Inc.",
+    "940ee7": "HUAWEI TECHNOLOGIES CO.,LTD",
```

Comment:
Noticing that maybe the ordering isn't working.  This is the same line as 6535 that is a remove. How is flatbot oui ordering this file, maybe we can get it fixed to avoid these "fake" updates. @qduk

Reply:
I actually wasn't doing any type of ordering. I made a commit on my #197 to order the dictionary before writing the file.

Reply:
cool thanks. Might as well merge this one than since it will be addressed in future flatbot PRs.